tests/__tests__/use-strict.spec.ts

```diff
@@ -11,7 +11,7 @@ describe('use strict', () => {
     const stderr = result.stderr;
 
     expect(result.status).toBe(1);
-    expect(stderr).toContain('Strict.ts:4:4');
+    expect(stderr).toContain('Strict.ts:4:3');
```

Comment:
this doesn't look right. The error is on the 4th line and not the 3rd - https://github.com/kulshekhar/ts-jest/blob/392551add1b5c1b1ba262079a42922b10632b94a/tests/use-strict/Strict.ts#L4

Reply:
Ignore this - I'm still half asleep. Didn't realize it was the column number